### DISCUSSION TITLE
Error updating from 1.32.2 > 1.32.3 (Cargo Build error)

### DISCUSSION THREAD

**Discussion by quantum-ciphers** [dani-garcia/vaultwarden#5168] [General]
Hi Everyone, 

Wondering if anyone has some pointers on how to get this error corrected. I have Vaultwarden deployed on an LXE container in Proxmox. Utilized the package found in the Helper Scripts, which I believe points to this source for updating. 

I run 'update' from the terminal and it appears to kick off but then fails about 2-3 minutes after initializing. Here is the error; 

[ERROR] in line 85: exit code 0: while executing command cargo build --features "sqlite,mysql,postgresql" --release &> /dev/null

Any help on getting this corrected would be appreciated.

**Comment by BlackDex** [comment]
I'm not sure which script you are taking about. It's probably a community provide script 

Unfortunately the line doesn't say anything useful except that it can't build Vaultwarden. The reason why it can't be build seems to be redirected to /dev/null.

**Reply by quantum-ciphers** [reply]
Thanks for the reply. Here is the script I'm using - https://tteck.github.io/Proxmox/#vaultwarden-lxc

I did manage to just spin up a new docker compose instance of vaultwarden and will be configuring that most likely. The maintainer of the helper scripts recently indicated that he will be stepping away from development projects so long-term the Proxmox Helper Scripts may not be the best to continue using anyways.

**Comment by nhalatek** [comment]
In my case, the error you mentioned was caused by a lack of space in the LXC container. To fix it, I simply had to increase the disk space for the container. The steps to do this from the Proxmox CLI are:

1. Find the container ID for Vaultwarden:
`pct list`
2. In my case, the container ID was 103, so I increased the storage space by 2GB:
`pct resize 103 rootfs +2G`

**Comment by Zobber** [comment]
Hi,

I got same error with Vaultwarden LXC in Proxmox with project "Proxmox Helper Scripts", in my case I with user "root" works executing:

`rustup update`

And try again the app update with same user:

`update`



**NOTE:** Verify storage space from steps by @nhalatek if its doesnt work
